.yamato/hdrp-osx_apple_silicon-metal.yml

```diff
@@ -0,0 +1,83 @@
+
+# This file is generated by ant_yaml_generation package, don't edit this file manually.
+# Introduce any changes under .yamato/config/*.metafile files and rerun yaml-generation (or check #devs-automation-tooling).
+
+{% metadata_file .yamato/_cache_config.metafile -%}
+{% metadata_file .yamato/_latest_editor_versions_trunk.metafile -%}
+
+---
+
+HDRP_OSX_Apple_Silicon_Metal_Standalone_mono_Linear_trunk:
+    name: HDRP on OSX_Apple_Silicon_Metal_Standalone_mono_Linear on version trunk
+    agent:
+        type: Unity::metal::macmini
+        image: slough-ops/macos-11-base:stable
+        model: M1
+        flavor: m1.mac
+    variables:
+        UPM_REGISTRY: https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates
+        CUSTOM_REVISION: '{{trunk.changeset.id}}'
+        CACHE_ARGS: '{{cache.flags}}'
+        UTR_VERSION: "current"
+        TEST_FILTER: .*
+    commands:
+      -  git clone git@github.cds.internal.unity3d.com:unity/com.unity.testframework.graphics.git TestProjects/HDRP_RuntimeTests/Packages/com.unity.testframework.graphics
+      -  git -C TestProjects/HDRP_RuntimeTests/Packages/com.unity.testframework.graphics checkout add-apple-silicon-support-2
```

Comment:
Is this line still needed? Maybe we should first promote the gfx test framework with the change to avoid checking out this specific branch?

Reply:
edit: seems it was promoted to 7.8.17 with the changes so I guess it's safe to remove that line :)

Reply:
Good catch, indeed, will push a fix!

Reply:
Kicked off these again to be sure
https://unity-ci.cds.internal.unity3d.com/job/10042195 
https://unity-ci.cds.internal.unity3d.com/job/10042194